tests/EvmAcceptanceTests/contracts/proxy/ImplementationV1.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity 0.8.9;
+
+import "./ProxyStorageV1.sol";
+import "./interfaces/InterfaceV1.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+contract ImplementationV1 is ProxyStorage, Initializable, OwnableUpgradeable, UUPSUpgradeable, InterfaceV1
```

Comment:
Contract names in hardhat are global, so probably worth giving this a more self-explanatory name (though I admit mine are just as bad!)